src/templates/components/staffProfileTeaser/index.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react'
+import { StaffProfileTeaser as FormattedStaffProfileTeaser } from '@/types/formatted/staffProfile'
+import { MediaImage } from '@/templates/common/mediaImage'
+import { PhoneNumber } from '@/templates/common/phoneNumber'
+
+export const StaffProfileTeaser = ({
+  media,
+  firstName,
+  lastName,
+  suffix,
+  description,
+  vamcTitle,
+  phoneNumber,
+  link,
+  id,
+}: FormattedStaffProfileTeaser) => {
+  return (
+    <div className="vads-u-display--flex vads-u-margin-bottom--4">
+      {media && (
+        <div
+          className="vads-u-margin-right--2 vads-u-flex--auto"
+          data-testid="staff-profile-image"
+        >
+          <MediaImage
+            {...media}
+            className="person-profile-detail-page-image vads-u-width--auto"
+            imageStyle="2_3_medium_thumbnail"
+          />
+        </div>
+      )}
+      <div>
+        {link ? (
+          <va-link
+            class="vads-u-display--block vads-u-margin-bottom--1 vads-u-font-family--serif vads-u-font-weight--bold vads-u-font-size--lg"
+            href={link}
+            text={`${firstName} ${lastName} ${suffix}`}
```

Comment:
Since there is space between line 16 and 17, maybe the common classes and title text can be shared between these two branches by putting them in a couple constants above?

Reply:
good idea!